extras/src/main/java/pl/project13/scala/jmh/extras/profiler/AsyncProfiler.java

```diff
@@ -144,7 +161,8 @@ private void createOutputDirectories() {
     public void beforeIteration(BenchmarkParams benchmarkParams, IterationParams iterationParams) {
         if (!started && iterationParams.getType() == IterationType.MEASUREMENT) {
             String threadOpt = this.threads ? ",threads" : "";
-            profilerCommand(String.format("start,event=%s%s,framebuf=%d,interval=%d", event, threadOpt, framebuf, interval));
+            String jfrOpt = this.jfr ? ",jfr,file=" + jfrFile().toAbsolutePath().toString() : "";
```

Comment:
`jfrFile()` here throws a `NullPointerException` when `output` is not set by the user, because the creation of a temporary output directory is relegated to `afterIteration`.

Reply:
Oops, I've pushed a fix.